3p/nameframe.max.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Semantic nit: Except for the developer-facing error message, which by ancient convention doesn't count, this file contains no English-language content and so a `lang` attribute is not appropriate.